src/components/Link/Link.js

```diff
@@ -37,7 +38,9 @@ export default class Link extends React.Component {
 Link.propTypes = {
   title: PropTypes.string,
   size: PropTypes.oneOf(['xs', 'sm', 'md', 'lg', 'xl']),
+  icon: PropTypes.string,
   disabled: PropTypes.bool,
+  position: PropTypes.oneOf(['left', 'right']),
```

Comment:
```suggestion
  iconPosition: PropTypes.oneOf(['left', 'right']),
```
Can we call it Icon Position?